arch/arm/core/cortex_m/reset.S

```diff
@@ -78,6 +78,11 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
     ldr r2, =CONFIG_ISR_STACK_SIZE
     bl memset
 #endif
+#ifdef CONFIG_STACK_SENTINEL
+    ldr r0, =_interrupt_stack
+    ldr r1, =0xF0F0F0F0
```

Comment:
this doesn't accomplish anything.

_check_stack_sentinel() looks for the stack sentinel for the currently running thread.
The interrupt stack isn't associated with any thread, so unless I'm missing something, this is never checked.

Reply:
There is a bit of a catch 22 on the debug support around stacks. In the Kernel shell there is a "stacks" command that will show the usage of all of the stacks. The problem is that it will show "0" if you are using the sentinel check. So I fixed that in the other file change but the stacks shell command is checking all of the the stacks including the interrupt stack so it needs to be an "all or none" approach to setting the sentinel even though there is no other place that checks to see if the interrupt stack has been blown.